src/main/scala/zio/redis/options/Strings.scala

```diff
@@ -0,0 +1,39 @@
+package zio.redis.options
+
+trait Strings {
+
+  sealed trait BitFieldType
+  object BitFieldType {
+    sealed case class UnsignedInt(size: Int) extends BitFieldType
```

Comment:
This isn't completely safe as it allows creation of invalid values, e.g. `UnsignedInt(1000)`. Instead, I'd use smart constructors here, we'll work on usability later.

Reply:
Could you please elaborate on this a little more? Using smart constructors would require error handling, which we have not considered so far.

Reply:
That's also true, and perhaps out of scope. We can note this one down and let it "leak" inside the error channel of response.